www/src/pages/nl/deployment/index.astro

```diff
@@ -0,0 +1,25 @@
+---
+import IndexPage from "../../../components/docs/indexPage.astro";
+import { SIDEBAR, type Frontmatter } from "../../../config";
+import { getLanguageFromURL } from "../../../languages";
+import Layout from "../../../layouts/docs.astro";
+
+const frontmatter: Frontmatter = {
+  title: "Uitrollen",
+  layout: "docs",
+  description: "Learn hoe je je T3-app kan uitrollen naar production.",
```

Comment:
> or keep it English because you could reason it being a technical term

I initially assumed that *production* should remain untranslated. However, upon quickly researching this, I found that several products and organizations do translate it.

- [Atlassian](https://www.atlassian.com/nl/agile/software-development/release)
- [Asana](https://asana.com/nl/resources/release-management)
- [Google Play Console](https://support.google.com/googleplay/android-developer/answer/9859348?hl=nl)

I’m sharing these examples to support the decision to translate it, in case anyone has concerns.